doc/user/README.md

```diff
@@ -19,6 +19,36 @@ For help contributing to the docs, see [`CONTRIBUTING.md`](./CONTRIBUTING.md).
 
 ## Tasks
 
+### (Temporary) Updating LTS Docs
+
+As we start to break apart the binary, we have to work through a temporary phase where some users will need the 0.26 binary docs and some will need the distributed-mode docs. For now, https://materialize.com/docs/ is the binary and https://materialize.com/docs/unstable/ is distributed mode.
+
+#### Shipping updates to Unstable Docs
```

Comment:
Shouldn't we always open the PRs against `main` and then backport if it _also_ applies to `lts-docs`? If so, we could cut this section out, but I could be missing some scenarios you thought of!

Reply:
I figure there might be some edge cases as we get closer and start rewriting high-level pages and quick starts. Like where `main` is rewritten and then we find a typo in `lts-docs`

Reply:
👍